tests/test_solver.py

```diff
@@ -121,40 +124,66 @@ def _submit_and_check(self, solver, linear, quad, **kwargs):
         return results
 
 @unittest.skipUnless(config, "No live server configuration available.")
-class Utilities(_QueryTest):
+class Utilities(unittest.TestCase):
     """Test solver utility methods."""
 
-    def test_qpu_access_time_estimate(self):
-        with Client(**config) as client:
-            solver = client.get_solver(qpu=True)
-
-            runtime_q1000 = solver.estimate_qpu_access_time(num_qubits=1000)
+    def test_qpu_access_time_estimate_mock(self):
+        solver_mock = StructuredSolver(data=mocks.qpu_access_time_data(), client=None)
+
+        # not allowed annealing_time together with anneal_schedule
+        with self.assertRaises(ValueError):
+            solver_mock.estimate_qpu_access_time(num_qubits=1000,
+                                            anneal_schedule=[[0.0, 0.0], [50.0, 0.5]],
+                                            annealing_time=150)
```

Comment:
You can split subtests into separate tests (initialize `solver_mock` in `setUp`/`setUpClass`), or use [`subTest`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.subTest). I'd vote for former.

Reply:
Implemented with ``setUpClass``